### PR TITLE
How it works URL clarification

### PR DESCRIPTION
**Pull request by CathPag**
Per @jihoon-seo's comment in [this issue](https://github.com/cncf/glossary/issues/799).
Jihoon, is - ok or should I change to _?

### CODE REVIEW

**Review by iamNoah1 — APPROVED** [review]
lgtm

File: content/en/contribute/_index.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Copy the content…
...
+Add the term name in the URL (no capitalization, no spaces, and use hyphens to separate words) and .md at the end (note: if your preview doesn't work, you probably forgot to add .md at the end). Now paste the template content below. Copy and paste your definition into the file. To make reviews easier, please **use [semantic line breaks](https://sembr.org/)** (e.g. one line per sentence). Note that GitHub uses markdown to format the text (e.g., hyperlink, bold, italic). Please refer to this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/). To verify you've used markdown as intended, go to "preview."
```
Comment (seokho-son) [inline content/en/contribute/_index.md:86]:
```suggestion
Add the term name in the URL (no capitalization, no spaces, no parentheses, and use hyphens to separate words) and .md at the end (note: if your preview doesn't work, you probably forgot to add .md at the end). Now paste the template content below. Copy and paste your definition into the file. To make reviews easier, please **use [semantic line breaks](https://sembr.org/)** (e.g. one line per sentence). Note that GitHub uses markdown to format the text (e.g., hyperlink, bold, italic). Please refer to this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/). To verify you've used markdown as intended, go to "preview."
```

**Review by seokho-son — COMMENTED** [review]
I added no parentheses :)

to prevent filenames like [TLS(Transport Layer Security).md](https://github.com/cncf/glossary/blob/main/content/en/TLS(Transport%20Layer%20Security).md)

**Review by iamNoah1 — APPROVED** [review]
lgtm

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Comment by iamNoah1** [comment]
@CathPag can you please resolve the merge conflicts :)

**Comment by jihoon-seo** [comment]
@iamNoah1 @CathPag I handled the merge conflict. 😊

**Review by iamNoah1 — APPROVED** [review]
lgtm

**Comment by iamNoah1** [comment]
thanks @jihoon-seo

**Pull request merged**